apicapi/tests/unit/common/test_apic_common.py

```diff
@@ -15,10 +15,14 @@
 #
 # @author: Henry Gessau, Cisco Systems
 # @author: Ivar Lazzaro (ivar-lazzaro), Cisco Systems Inc.
+# @author: Amit Bose (amibose@cisco.com), Cisco Systems Inc.
 
 import contextlib
 import mock
 import requests
+import tempfile
+from OpenSSL import crypto
```

Comment:
this should go after mock  and base64 on top of the import list (per first comment)